pkg/client/factory.go

```diff
@@ -68,6 +69,7 @@ func NewFactory(baseName string) Factory {
 
 	f.flags.StringVar(&f.kubeconfig, "kubeconfig", "", "Path to the kubeconfig file to use to talk to the Kubernetes apiserver. If unset, try the environment variable KUBECONFIG, as well as in-cluster configuration")
```

Comment:
This part of the message isn't clear:
"If unset, try the environment variable KUBECONFIG, as well as in-cluster configuration"
Are these meant to be alternatives to setting the path? Do you need both? Is either of them the default if not set? Or would you have to try one or the other for the command to succeed? In other words, does not specifying result in an error condition?
Or does the command look for a KUBECONFIG environment variable, and if not found then look for the path in the cluster config?
If it's the latter, the fix is fairly simple:
"If unset, tries the environment variable KUBECONFIG, and if not present, tries in-cluster configuration"
If not the latter, questions ^^ might help with clearer text.

Reply:
@Bradamant3 we can address the `kubeconfig` text in another issue. This PR is about `kubecontext` 😄